README.adoc

```diff
@@ -321,7 +362,7 @@ You can check the health of the application by running the following command:
 [source,bash]
 ----
 $ curl localhost:8080/actuator/health
-{"status":"UP"}
+{"status":"UP","groups":["liveness","readiness"]}
```

Comment:
I'm not sure that groups always show up. I think they only appear on Kubernetes or if enabled.

Reply:
They do appear when running the lab in the interactive environment (spring-guides repo version)